aten/src/ATen/Context.cpp

```diff
@@ -147,6 +147,21 @@ void Context::setAllowTF32CuBLAS(bool b) {
   allow_tf32_cublas = b;
 }
 
+at::LinalgBackend Context::linalgPreferredBackend() const {
+  return linalg_preferred_backend;
+}
+
+void Context::setLinalgPreferredBackend(at::LinalgBackend b) {
+  linalg_preferred_backend = b;
+  if (b != at::LinalgBackend::Default) {
+    TORCH_WARN_ONCE(
+      "torch.backends.cuda.preferred_linalg_library is an experimental feature. "
+      "If you see any error or regression when this flag is set "
```

Comment:
Perf regressions are expected if you pick the wrong backend, right? So the error message shouldn't suggest filing issue in this case. It could suggest filing an issue for errors or unexpected behaviors.

Reply:
Thanks for the suggestion. I've changed the warning message.